src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigFileGenerator.cs

```diff
@@ -52,6 +54,12 @@ internal static partial class EditorConfigFileGenerator
             return editorconfig.ToString();
         }
 
+        internal static IEnumerable<(string feature, ImmutableArray<IOption2> options)> GetLanguageAgnosticEditorConfigOptions()
+        {
+            yield return (WorkspacesResources.Core_EditorConfig_Options, FormattingOptions2.Options);
+            yield return (WorkspacesResources.dot_NET_Coding_Conventions, GenerationOptions.AllOptions.AddRange(CodeStyleOptions2.AllOptions));
+        }
```

Comment:
I wouldn't place this helper here. It's unnecessary adding dependency to formatting and generation options to a type that doesn't need to have that dependency.

Reply:
Moved to the EditorConfigOptionsGenerator